src/main/resources/static/js/axios.min.js

```diff
@@ -0,0 +1,9 @@
+//# sourceMappingURL=axios.min.map
```

Comment:
new line

Reply:
This is an external file reference, we better not change it.